### ISSUE TITLE
Select mode, multiple cursor mode and jump after mode not working

### ISSUE THREAD

**Issue by superxiao** [ice9js/ace-jump-sublime#60]
Sublime Text Build 3126:

I tried in a Windows 7 machine and none of the above modes can be activated with the default key bindings. Changing key bindings does not fix the issue too. Later I tried in a MacBook and the above modes cannot be activated too.

Are these modes currently working?

**Comment by robinrosenstock** [comment]
For me neither. Not on Windows 10 nor on Ubuntu 16.

**Comment by carlton87** [comment]
Same problem here (Windows 10).

**Comment by xieguo** [comment]
Sublime Text Build 3126:  Select mode, multiple cursor mode and jump after mode not working on windows 10

**Comment by kurtontheway** [comment]
These modes are also not working here on latest macOS and Centos 7, Sublime Text Build 3126.

**Comment by OmanF** [comment]
Those modes **do** work but you have to "activate" them **after** "firing up" the AceJump functionality.
I.e. suppose you want to do a _select mode_ word-jump (assuming default key bindings), you **first** "fire up" the word-jump (default: ctrl+shift+;) then, **while the AceJump bar is open at the bottom of the screen** you "fire up" the select-jump (default: alt+;), **then** you enter the head character and select the label.

Same goes for add-cursor and jump-after. They all work, but it's not intuitive (and not complying with the key binding **notation** of other Sublime Text's plugins that have two-step activation).

I vote to close the issue.

**Comment by robinrosenstock** [comment]
Thanks @OmanF. This should be mentioned in the Readme! Maybe you want to make a little Pull Request for the Readme @OmanF?

**Comment by OmanF** [comment]
Thanks, @geniusupgrader.
Submitted a PR.

**Comment by ice9js** [comment]
Thanks for clearing this one up @OmanF.  
It's good to hear you found a work around but I'm a little hesitant on closing the issue just yet as that's actually not the intended behaviour.

The intended behaviour is for the *modifier* to be activated **before** *firing up* the jump - and that's what has worked for me on every machine I could get my hands on - hence the issue remaining open for so long.

You mentioned the key bindings are not complying with the notation of other ST plugins. Would you mind sharing a source ? I'd definitely be interested if there are better ways to do things.

I still think it's worth adding the note to the Readme - but maybe more in the style of known issues ?

**Comment by OmanF** [comment]
Hi @ice9js,

- No source per-se, but all mutli-binding plugins I use conform to the notation: `[bind A, bind B,...]`. So, even with the intended behavior the de-facto notation would have been: `[ctrl+shift+;, alt+.]` (for example).
- As for the `README`... I'd more than be happy to edit my PR and move the comments to a _Known Issue_ section. I'm guessing at the end of the file? I'll do it later today when I've got some spare time and resubmit the PR.

BTW, for debugging purposes, I've tested the plugin on both Fedora 26 and Antergos Linux (which I keep up to date on daily basis), both using ST build 3126... the workaround was the way to make it work, not the intended way.

All in all, thanks for porting one of Emacs's best features, **successfully**. A huge time saver. Kudos!
